Hunting Queries/SecurityAlert/AlertsWithFile.yaml

```diff
@@ -0,0 +1,29 @@
+id: 11d808a1-32fe-4618-946a-cfd43523347a
+name: Alerts related to File
+description: |
+  'Any Alerts that fired related to a given File during the range of +6h and -3d'
+requiredDataConnectors:
+  - connectorId: AzureSecurityCenter
+    dataTypes:
+      - SecurityAlert
+  - connectorId: MicrosoftCloudAppSecurity
+    dataTypes:
+      - SecurityAlert
+tactics:
+  - Persistence
+  - Discovery
+  - LateralMovement
+  - Collection
+query: |
+
+  let GetAllAlertsWithFile = (suspiciousEventTime:datetime, v_File:string){
+  let v_StartTime = suspiciousEventTime-1d;
+  let v_EndTime = suspiciousEventTime+1d;
+  SecurityAlert
+  | where TimeGenerated between (v_StartTime .. v_EndTime)
+  | extend Computer = toupper(parse_json(ExtendedProperties).["Compromised Host"])
+  | where ExtendedProperties has v_File
+  | extend timestamp = TimeGenerated, HostCustomEntity = Computer
```

Comment:
consider adding IP and User entities if present in extended properties or entities columns?

Reply:
done